### PR TITLE
E2E to v1.0

### PR DESCRIPTION
**Pull request by nachocano**
## Proposed Changes

- E2E tests with 1.0
- Updating CE sdk for this bug fix https://github.com/cloudevents/sdk-go/pull/272

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -63,7 +63,7 @@ required = [
 
 [[override]]
   name = "github.com/cloudevents/sdk-go"
-  version = "0.10.0"
+  branch = "master"
```
Comment (grantr) [inline Gopkg.toml]: Can you add a comment explaining why this is master and when it can be moved back to a released version?

Comment (nachocano) [reply Gopkg.toml]: sure, my bad. Let me add that in a follow up... Thanks!

Comment (nachocano) [reply Gopkg.toml]: Done! https://github.com/google/knative-gcp/pull/449

**Pull request merged**